provider/runbook_data.go

```diff
@@ -35,6 +38,15 @@ func dataSourceRunbook() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"attachment_rule": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsJSON),
+				StateFunc: func(value interface{}) string {
+					normalizedJSON, _ := structure.NormalizeJsonString(value)
+					return normalizedJSON
+				},
+			},
```

Comment:
since the user can't specify this field in a data source, this should be computed and then we can get rid of the validate func.
```suggestion
			"attachment_rule": {
				Type:             schema.TypeString,
				Computed         true,
				StateFunc: func(value interface{}) string {
					normalizedJSON, _ := structure.NormalizeJsonString(value)
					return normalizedJSON
				},
			},
```

Reply:
ah, okay this makes sense I was curious why this would be needed in the data side. Curious where you would use a data runbook at.